nhais/README.md

```diff
@@ -0,0 +1,40 @@
+# NHAIS Adaptor - Developer Notes
+The following sections are intended to provide the necessary info on how to configure and run the NHAIS adaptor.
+
+## Pre-requisites
+
+Ensure you have Pipenv installed and on your path, then within nhais directory, run:
+
+`pipenv install`
+
+## Developer setup 
+
+Start IntelliJ
```

Comment:
I'm not sure that IntelliJ instruction should be placed at git repository. It's open-source project so developers might use whatever tool they want

Reply:
Removed all intelliJ references